### PR TITLE
[11.2] Add missing parameters to the `GitLab\Api\Projects:all` method

### LINKED ISSUE #622: Projects:all parameter list is incomplete
When comparing the current list of attributes of the [`/projects` API endpoint](https://docs.gitlab.com/ee/api/projects.html#list-all-projects) to the [parameter list of the `\Gitlab\Api\Projects::all` class](https://github.com/GitLabPHP/Client/blob/11.1/src/Api/Projects.php#L25-L42), the following parameters are missing:
* `id_after` (integer)
* `id_before` (integer)
* `last_activity_after` (datetime)
* `last_activity_before` (datetime)
* `repository_checksum_failed` (boolean)
* `repository_storage` (string)
* `wiki_checksum_failed` (boolean)
* `with_custom_attributes` (boolean)
* `with_programming_language` (string)

### PR DESCRIPTION
**Pull request by aadmathijssen**
The following missing parameters are added to the `\GitLab\Api\Projects:all` method:
* `id_after` (integer)
* `id_before` (integer)
* `last_activity_after` (datetime)
* `last_activity_before` (datetime)
* `repository_checksum_failed` (boolean)
* `repository_storage` (string)
* `starred` (boolean)
* `wiki_checksum_failed` (boolean)
* `with_custom_attributes` (boolean)
* `with_programming_language` (string)

---

Closes #622.

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Thanks for this. Could you update the method phpdoc too please, and add a change log entry? :)

**Comment by aadmathijssen** [comment]
Done!

I also noticed that the `starred` parameter was already there, so I remove the duplicate addition from the PR.

File: src/Api/Projects.php [hunk 1]
```diff
@@ -129,10 +139,6 @@ public function all(array $parameters = [])
             ->setNormalizer('repository_checksum_failed', $booleanNormalizer)
         ;
         $resolver->setDefined('repository_storage');
-        $resolver->setDefined('starred')
```
Comment (GrahamCampbell) [inline src/Api/Projects.php]: ?

File: src/Api/Projects.php [hunk 2]
```diff
@@ -40,6 +41,15 @@ class Projects extends AbstractApi
      *     @var bool   $with_issues_enabled         limit by enabled issues feature
      *     @var bool   $with_merge_requests_enabled limit by enabled merge requests feature
      *     @var int    $min_access_level            Limit by current user minimal access level
+     *     @var int    $id_after                    Limit by project id's greater than the specified id
+     *     @var int    $id_before                   Limit by project id's less than the specified id
+     *     @var DateTimeInterface $last_activity_after  Limit by last_activity after specified time
```
Comment (GrahamCampbell) [inline src/Api/Projects.php]: Please keep this aligned

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [11.2.0] - UPCOMING
 
 * UPCOMING
```
Comment (GrahamCampbell) [inline CHANGELOG.md]: Please remove this bullet, replacing it with yours.

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: src/Api/Projects.php (hunk 1 above)
Comment (aadmathijssen) [reply src/Api/Projects.php]: This was accidentally added: the 'starred' parameter was already supported.

File: CHANGELOG.md (hunk 3 above)
Comment (aadmathijssen) [reply CHANGELOG.md]: Done.

File: src/Api/Projects.php (hunk 2 above)
Comment (aadmathijssen) [reply src/Api/Projects.php]: Done.

**Pull request merged**